TicketsDemo/Controllers/RunController.cs

```diff
@@ -68,7 +73,11 @@ public class RunController : Controller
         [HttpPost]
         public ActionResult CreateTicket(CreateTicketModel model)
         {
-            var tick = _tickServ.CreateTicket(model.ReservationId,model.FirstName,model.LastName);
+            var parameters = new PriceCalculationParameters
+            {
+                BookingServiceId = model.BookingServiceId
+            };
+            var tick = _tickServ.CreateTicket(model.ReservationId,model.FirstName,model.LastName, parameters);
```

Comment:
Functions shall never expect half-way-initialized DTO objects.

If some fields of DTO are not needed - create another DTO without these fields (in your case - just add BookingServiceId as int argument to function)

ALternatively, fill 'all' the properties of PriceCalculationParameters 'before' calling CreateTicket